CADETProcess/processModel/flowSheet.py

```diff
@@ -874,7 +789,7 @@ def add_product_outlet(self, product_outlet):
             raise CADETProcessError('Expected Outlet')
         if product_outlet in self._product_outlets:
             raise CADETProcessError(
-                f'Unit \'{product_outlet}\' is already a product outlet'
+                f"Unit '{product_outlet}' is already a product outlet"
```

Comment:
Separate code formatting and the improved flowrate calculation into different commits. Maybe even ignore it (for now). Probably we should format the entire code base at once but this should also be a different issue/PR